### ISSUE TITLE
iOS17.4系统一些列表点击cell无响应、崩溃

### ISSUE THREAD

**Issue by sunshinearea** [tigerAndBull/TABAnimated#235]
比较了两个系统，ios15系统均正常响应，ios17系统列表多次复现，但是去掉骨架屏加载点击正常，麻烦作者看下

### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS 
 Platform Version        | e.g. 17.4.1 
 TABAnimated Version      | e.g. 2.6.5
 Xcode Version           | e.g. Xcode 15
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
提供下crash堆栈

**Comment by sunshinearea** [comment]
> 提供下crash堆栈
* thread #1, stop reason = breakpoint 2.1
    frame #0: 0x000000010340987e hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:491:9
  * frame #1: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #2: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #3: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #4: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #5: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #6: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #7: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #8: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #9: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #10: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #11: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #12: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #13: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #14: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #15: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #16: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #17: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #18: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #19: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #20: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #21: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #22: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #23: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #24: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #25: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #26: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #27: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #28: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #29: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #30: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #31: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #32: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #33: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #34: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #35: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #36: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #37: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #38: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #39: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #40: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #41: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #42: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #43: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #44: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #45: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #46: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #47: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #48: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #49: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tab_tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #50: 0x00000001034098e4 hcezt`-[TABTableAnimated tab_tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at TABTableAnimated.m:492:9
    frame #51: 0x0000000102d73954 hcezt`+[SADelegateProxy invokeWithTarget:selector:](self=SAExposureDelegateProxy, _cmd="invokeWithTarget:selector:", target=0x0000600002666c70, selector="tableView:willDisplayCell:forRowAtIndexPath:") at SADelegateProxy.m:126:9
    frame #52: 0x0000000103302fa5 hcezt`-[SAExposureDelegateProxy tableView:willDisplayCell:forRowAtIndexPath:](self=0x0000600002666c70, _cmd="tableView:willDisplayCell:forRowAtIndexPath:", tableView=0x00007fe09505c400, cell=0x00007fe0a4336a00, indexPath=0xcf6dce8106776dd3) at SAExposureDelegateProxy.m:51:9
    frame #53: 0x0000000133e4c916 UIKitCore`-[UITableView _notifyWillDisplayCell:forIndexPath:] + 115
    frame #54: 0x0000000133e4c489 UIKitCore`-[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:] + 1957
    frame #55: 0x0000000133e10bad UIKitCore`-[UITableView _updateVisibleCellsForRanges:createIfNecessary:] + 788
    frame #56: 0x0000000133e114b6 UIKitCore`-[UITableView _updateVisibleCellsNow:] + 1800
    frame #57: 0x0000000133e338f9 UIKitCore`-[UITableView layoutSubviews] + 205
    frame #58: 0x00000001342376af UIKitCore`-[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 2138
    frame #59: 0x00000001145f92bd QuartzCore`CA::Layer::layout_if_needed(CA::Transaction*) + 527
    frame #60: 0x00000001146050e9 QuartzCore`CA::Layer::layout_and_display_if_needed(CA::Transaction*) + 67
    frame #61: 0x0000000114506334 QuartzCore`CA::Context::commit_transaction(CA::Transaction*, double, double*) + 706
    frame #62: 0x00000001145407ce QuartzCore`CA::Transaction::commit() + 728
    frame #63: 0x0000000114541cd2 QuartzCore`CA::Transaction::flush_as_runloop_observer(bool) + 60
    frame #64: 0x0000000116619308 CoreFoundation`__CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
    frame #65: 0x0000000116613c9e CoreFoundation`__CFRunLoopDoObservers + 534
    frame #66: 0x00000001166141e6 CoreFoundation`__CFRunLoopRun + 1158
    frame #67: 0x000000011661397d CoreFoundation`CFRunLoopRunSpecific + 557
    frame #68: 0x000000012375f08f GraphicsServices`GSEventRunModal + 137
    frame #69: 0x0000000133b6553d UIKitCore`-[UIApplication _run] + 972
    frame #70: 0x0000000133b69fab UIKitCore`UIApplicationMain + 123
    frame #71: 0x0000000103510e78 hcezt`main(argc=1, argv=0x000000030e2eeb70) at main.m:14:16
    frame #72: 0x00000001108023e0 dyld_sim`start_sim + 10
    frame #73: 0x0000000208130386 dyld`start + 1942

**Comment by sunshinearea** [comment]
你好上面这个是神策那个问题的，第一个问题我提问后发现了问题补充了说明您看下

**Comment by tigerAndBull** [comment]
提供下完整堆栈

**Comment by sunshinearea** [comment]
> 提供下完整堆栈
问题一：点击时判断如果在动画中就不执行下面的点击操作，我加了判断如果在动画中先清除再开启动画解决了问题
问题二：
[exception.txt](https://github.com/tigerAndBull/TABAnimated/files/15085182/exception.txt)
